src/phoenix/db/migrations/versions/cf03bd6bae1d_init.py

```diff
@@ -77,6 +77,41 @@ def upgrade() -> None:
         sa.Column("cumulative_llm_token_count_prompt", sa.Integer, nullable=False),
         sa.Column("cumulative_llm_token_count_completion", sa.Integer, nullable=False),
     )
+
+    op.create_table(
+        "span_annotations",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column("span_rowid", sa.Integer, sa.ForeignKey("traces.id"), nullable=False),
```

Comment:
```suggestion
        sa.Column("span_rowid", sa.Integer, sa.ForeignKey("spans.id"), nullable=False),
```

Reply:
addressed in c321357b38a19795186658d908a07581f6ee8905